### PR TITLE
Adding an easy way to install pip on Mac OS X 

### PR DESCRIPTION
**Pull request by GregBorrelly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Note: none of your data is modified nor sent anywhere during execution
 # Dependencies easy install
 If you don't have pip, see https://pip.pypa.io/en/stable/installing/
 
+Another option on Mac OS X is to use `sudo easy_install pip`
```
Comment (mdezube) [inline README.md]: I like this suggestion, but how about just rephrasing line 27 as:

If you don't have pip, see https://pip.pypa.io/en/stable/installing/, or if using a Mac run `sudo easy_install pip`

Comment (mdezube) [inline README.md]: I like it!  But how about changing line 27 to:

If you don't have pip, see https://pip.pypa.io/en/stable/installing/, or if using a Mac run `sudo easy_install pip`

**Review by mdezube — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GregBorrelly) [reply README.md]: Making Changes now !!! thank you.

Comment (GregBorrelly) [reply README.md]: Made the changes. Should be good to go now.

**Pull request merged**